aten/src/ATen/CPUGeneral.cpp

```diff
@@ -0,0 +1,14 @@
+#include <ATen/CPUGeneral.h>
+
+namespace at {
+size_t num_threads = 0;
+std::shared_ptr<tbb::task_scheduler_init> tbbinit;
+void set_num_threads(size_t num_threads_) {
+  if (num_threads == 0) {
```

Comment:
So we can only initialize the number of threads once?

Reply:
Hm, let me look into that.